### ISSUE TITLE
Total connection time incorrect

### ISSUE THREAD

**Issue by SlimeDog** [Rsl1122/Plan-PlayerAnalytics#400]
Spigot 1.12.2 latest
Plan 4.0.4

![untitled](https://user-images.githubusercontent.com/17748923/32259685-657b783e-be7f-11e7-99df-e5be2541de57.png)

config.yml
```
...
Customization:
  UseServerTime: true
  Display:
    SessionsAsTable: true
  Formatting:
    DecimalPoints: '#.##'
    TimeAmount:
      Year: '1 year, '
      Years: '%years% years, '
      Month: '1 month, '
      Months: '%months% months, '
      Day: '1 day '
      Days: '%days% days '
      Hours: '%zero%%hours%:'
      Minutes: '%hours%%zero%%minutes%:'
      Seconds: '%minutes%%zero%%seconds%'
      Zero: 00:00
...
```
Same incorrect display in other locations.

**Comment by Rsl1122** [comment]
Seems like month addition broke it

**Comment by SlimeDog** [comment]
Something in my config, or in code?

**Comment by SlimeDog** [comment]
Confirmed fix in Plan 4.0.5

**Comment by Rsl1122** [comment]
Cheers
